app/modules/teams/resources.py

```diff
@@ -8,19 +8,21 @@
 import logging
 
 from flask_restplus import Resource
+from flask_restplus_patched._http import HTTPStatus
 
 from app.extensions import db
 from app.extensions.api import Namespace, abort, http_exceptions
 from app.extensions.api.parameters import PaginationParameters
 from app.modules.users import permissions
 from app.modules.users.models import User
 
+
```

Comment:
There is no need in this extra space line, but I will merge and we just fix this later.